database/certificate.go

```diff
@@ -388,6 +388,38 @@ func (db *DB) GetAllCertsInStore(store string) (out []certificate.Certificate, e
 	}
 }
 
+func (db *DB) GetAllCertIDsIssuedBy(sha256 string)(out []int64, err error) {
```

Comment:
nit: the naming convention for DB function contains the hash type, so something like `GetAllCertIDsBySHA256Fingerprint`.